styles/components/_search.scss

```diff
@@ -232,7 +232,10 @@
     @include ktheme() {
       border-bottom-color: theme('placeholder-color');
       background: theme('background-color');
-      color: theme('text-color');
+      color: theme('text-color') !important;
+      &:hover {
+        background: theme('background-color') !important;
+      }
```

Comment:
They are the same change as in other issue ? https://github.com/kodadot/nft-gallery/pull/5064/files